webui/react/src/pages/TaskList.tsx

```diff
@@ -121,6 +121,10 @@ const TaskList: React.FC = () => {
         getJupyterLabs({ signal: canceler.signal }),
         getShells({ signal: canceler.signal }),
         getTensorBoards({ signal: canceler.signal }),
+        // getCommands({ signal: canceler.signal, users: settings.user }),
+        // getJupyterLabs({ signal: canceler.signal, users: settings.user }),
+        // getShells({ signal: canceler.signal, users: settings.user }),
+        // getTensorBoards({ signal: canceler.signal, users: settings.user }),
```

Comment:
I would say since these are replaced, you can remove the commented lines from this file